host-interaction/bootloader/manipulate-boot-configuration.yml

```diff
@@ -0,0 +1,25 @@
+rule:
+  meta:
+    name: manipulate boot configuration
+    namespace: host-interaction/bootloader
+    author: william.ballenthin@fireeye.com
+    scope: function
+    references:
+      - https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/bcdedit-command-line-options
+    examples:
+      - 7FBC17A09CF5320C515FC1C5BA42C8B3:0x40CA00
+  features:
+    - or:
+      - or:
+        - string: /bcdedit.exe/i
+        - optional:
+          - string: "/deletevalue safeboot"
+      - and:
+        - string: /boot.ini/i
+        - optional:
+          - api: kernel32.GetPrivateProfileStringA
+          - api: kernel32.WritePrivateProfileString
+          - or:
+            - string: "default"
+            - string: "boot loader"
+            - string: "operating systems"
```

Comment:
these were just some random things i encountered in the example binary. i can remove/tweak if this rule isn't specific enough

Reply:
do we need to consider lower/upper case?

Reply:
good catch

![image](https://user-images.githubusercontent.com/156560/117012443-2d579900-acac-11eb-8989-c22fd8839d59.png)